### PR TITLE
Improvement: refresh status bar and related fix

### PR DESCRIPTION
**Pull request by randy3k**
Do not refresh status bar using `GsInterfaceFocusEventListener`, it is handled by `GsStatusBarEventListener` already.

### CODE REVIEW

File: core/commands/status_bar.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def run(self, edit):
         global last_execution, update_status_bar_soon
         if sublime.load_settings("GitSavvy.sublime-settings").get("git_status_in_status_bar"):
 
-            millisec = int(round(time.time() * 100))
+            millisec = int(round(time.time() * 1000))
             # If we updated to less then 100 ms we don't need to update now but
```
Comment (stoivo) [inline core/commands/status_bar.py]: Update the comment

Comment (randy3k) [inline core/commands/status_bar.py]: The comment is correct, the timeout is 100ms

Comment (stoivo) [inline core/commands/status_bar.py]: Ahh, and time.time() returns seconds. Thanks

**Comment by stoivo** [comment]
Are there any specific reasons why you add `refresh_status_bar`, just to reduce the calls?

**Comment by randy3k** [comment]
Both `GsStatusBarEventListener` and `GsInterfaceFocusEventListener` have a `on_activated` method to call `gs_update_status_bar`. This PR avoids duplicately calling to it.

**Pull request merged**